src/pages/index.js

```diff
@@ -87,6 +87,12 @@ export default function Home() {
           </h3>
         <div className={styles.heroHowItWorks}>
         <img className={styles.heroLogo} src={useBaseUrl('img/how-it-works.png')} alt="Karmada Control Plane" />
+        </div>
+        <h3 className="hero__subtitle">
+            <Translate>Supporters</Translate>
+          </h3>
+        <div className={styles.heroSupporter}>
+        <img className={styles.heroLogo} src={useBaseUrl('img/supporters.png')} alt="Karmada Supporters" />
```

Comment:
Are you putting all the supporters into one image? I don't think we should. I don't want to edit the image every time a new supporter joined.

That explains why the picture looks indistinct.

Reply:
In my opinion, the overall flow of the website looks good when we put all the supporters into one image. We could change that. Personally, I prefer combination into one image. Please, let me know what you think.

Reply:
Anything else that you would like to see on index page through this PR? @RainbowMango

Reply:
`why Karmada` section looks unordered?

Can we organize the content by the table?

Reply:
`Why Karmada` section is defined that way.
![Screenshot from 2022-04-18 17-57-01](https://user-images.githubusercontent.com/54068868/163806741-f6e999b7-1326-455b-b643-22fac7e68a4b.png)

Reply:
Can we add some dummy images provisionally? for good-looking.

Reply:
I tried doing that. It looks something like this:

![Screenshot from 2022-04-19 15-01-18](https://user-images.githubusercontent.com/54068868/163972026-9cc38fca-763c-4030-95f6-09489587c56c.png)

 We could use different image for each of them.

Reply:
That's OK. Please push it on this PR. I'll invite other people to comment on the preview site.

Reply:
Done.